### PR TITLE
More README updates + add event location

### PR DESCRIPTION
**Pull request by doitlikejustin**
Went through and made some more detailed instructions on your README! I saw that was an item in your todo 😁

I was not too familiar with `clasp` before working with peloCal so I added some basic instructions for others who might not be familiar either.

Also, cleaned up & commented a few things in the code (nothing major) and added a feature to add an event location (i.e. Peloton) on newly created events.

If you see anything you'd like me to make changes on before merging this PR just let me know.

Thanks!

### CODE REVIEW

File: appsscript.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+  "timeZone": "America/Los_Angeles",
   "dependencies": {},
   "exceptionLogging": "STACKDRIVER",
   "runtimeVersion": "V8",
   "webapp": {
     "executeAs": "USER_DEPLOYING",
     "access": "MYSELF"
   }
-}
+}
```
Comment (delikat) [inline appsscript.json:10]: nit: newline

File: README.md [hunk 2]
```diff
@@ -4,20 +4,41 @@ A simple script to synchronize your Peloton schedule with Google Calendar. peloC
...
+1. Download peloCal repo locally
+2. Install `clasp` via `npm install @google/clasp -g` ([full setup instructions here](https://developers.google.com/apps-script/guides/clasp))
+3. Login to Google Apps Script with `clasp login`
+4. Clone the peloCal project from Google Apps Script `clasp clone <scriptId>`. To get your project Script ID:
+	1. Open Apps Script project.
+	2. At the left, click **Project Settings**
+	3. Under IDs, copy the **Script ID**.
+4. Note: Once the project is cloned, Google will convert the existing the `.gs` files in your Apps Script and download them locally as `.js` files. These can be removed, and we are only going to be using the `.ts` files in this repo.
+5. *...Make desired changes...*
+6. Push changes to Apps Script with `clasp push`
```
Comment (delikat) [inline README.md:31]: could you move this to a CONTRIBUTING.md file just to keep the README a bit cleaner for casual users?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "peloCal",
+  "version": "V1",
```
Comment (delikat) [inline package.json]: this field should be in semver format, let's set it to `1.0.0`

File: README.md [hunk 4]
```diff
@@ -4,20 +4,41 @@ A simple script to synchronize your Peloton schedule with Google Calendar. peloC
...
 
 ## Notes
 
 - Please be kind to the Peloton API and don't set too aggressive of a trigger (e.g. every minute), or you may find yourself rate limited.
 - This script interacts with _unofficial, undocumented_ endpoints, meaning it can and probably will break if Peloton changes their API.
 - Thanks to [@DovOps](https://github.com/DovOps) for his incredibly helpful [unofficial Swagger docs for the Peloton API](https://app.swaggerhub.com/apis/DovOps/peloton-unofficial-api/).
 
-## TODO / Feature Ideas
+## Todo
```
Comment (delikat) [inline README.md:39]: nit: i'd prefer to keep this as is, `TODO / Feature Ideas`

File: Code.ts [hunk 5]
```diff
@@ -31,6 +37,7 @@ function createEventFromRide(
     new Date(startDate.getTime() + ride.duration * 1000),
     {
       description: `${ride.description}\n\n${EVENT_DESCRIPTION_SIGNATURE}`,
+      location: `${EVENT_LOCATION}`
```
Comment (delikat) [inline Code.ts:40]: nice!

File: Code.ts [hunk 6]
```diff
@@ -16,13 +16,19 @@ const REMINDER_MINUTES_BEFORE = 5;
 
 // Exact name of the Google Calendar in which events will be created. Leave blank for default calendar
 const TARGET_CALENDAR_NAME = '';
+
 // =====================
 
+// Function to select under "Choose which function to run" in Triggers
+function run() {
+	main();
+}
```
Comment (delikat) [inline Code.ts]: spacing looks a little wonky here

```suggestion
function run() {
  main();
}
```

File: Code.ts [hunk 7]
```diff
@@ -89,9 +96,17 @@ function main() {
     .getEvents(now, endDate, { search: EVENT_DESCRIPTION_SIGNATURE })
     .forEach(event => {
       const id = event.getTag(EVENT_ID_TAG);
+      const title = event.getTitle();
+      const startTime = event.getStartTime();
       // delete calendar events for rides no longer on the schedule
-      if (!scheduledRideIds.includes(id)) {
-        console.log(`Deleting event for ride ${id}...`);
+      // check to see if the current start time is after current time
+      // ... since getEvents() will returns events that start during
```
Comment (delikat) [inline Code.ts]: typo

```suggestion
      // ... since getEvents() will return events that start during
```

File: Code.ts [hunk 8]
```diff
@@ -89,9 +96,17 @@ function main() {
...
+      // ... the given time range, ends during the time range, or 
+      // ... encompasses the time range. If the Apps Script trigger is set 
+      // ... to a shorter time (5-30 minutes) it will remove scheduled
+      // ... workouts that have just started but not eneded.
```
Comment (delikat) [inline Code.ts]: typo
```suggestion
      // ... workouts that have just started but not ended.
```

**Review by delikat — CHANGES_REQUESTED** [review]
Thank you so much for the PR, and apologies for the very very late reply! Really appreciate you adding the `clasp` instructions–super helpful.

The reorganization & comments look great, just a few minor code style nits / typos I'd like to clean up before merging.

File: appsscript.json (hunk 1 above)
Comment (doitlikejustin) [reply appsscript.json:10]: Not 100% clear on the issue here with the newline?

**Comment by doitlikejustin** [comment]
Everything is updated with the exception of that newline comment!

**Review by delikat — APPROVED** [review]

**Pull request merged**